tests/baselines/reference/booleanLiteralsContextuallyTypedFromUnion.errors.txt

```diff
@@ -0,0 +1,31 @@
+error TS2318: Cannot find global type 'CallableFunction'.
+error TS2318: Cannot find global type 'NewableFunction'.
+
+
+!!! error TS2318: Cannot find global type 'CallableFunction'.

```

Comment:
Probably shouldn’t get an error here.

Reply:
I didn't want to update the private lib.d.ts used by the tests as I suspect that'll cause some really noisy baseline changes.

Reply:
OK, just wanted to make sure this error wouldn't show up in some public-facing scenario.